config/config.go

```diff
@@ -334,6 +341,11 @@ var Default = Config{
 	AnalyticsConfig: AnalyticsConfigConfig{
 		IgnoredIPs: make([]string, 0),
 	},
+	DnsCache: DnsCacheConfig{
+		Enabled:       false,
+		TTL:           3600,
```

Comment:
better to move 3600 and 60 to constants

Reply:
fixed in 7f332e6a861e73fc8eb846e0e9aadd5f96fc724e